benchmarks/benchmarks/ag_methods.py

```diff
@@ -202,7 +202,94 @@ def time_wrap(self, num_atoms):
         atomgroup with default params.
         """
         self.ag.wrap()
-        
+
+    def time_center_of_mass_default(self, num_atoms):
+        """Benchmark center_of_mass calculation with
+        pbc and unwrap inactive.
+        """
+        self.ag.center_of_mass()
+
+    def time_center_of_mass_pbc(self, num_atoms):
+        """Benchmark center_of_mass calculation with
+        pbc active.
+        """
+        self.ag.center_of_mass(pbc=True)
+
+    def time_center_of_geometry_default(self, num_atoms):
+        """Benchmark center_of_geometry calculation with
+        pbc and unwrap inactive.
+        """
+        self.ag.center_of_geometry()
+
+    def time_center_of_geometry_pbc(self, num_atoms):
+        """Benchmark center_of_geometry calculation with
+        pbc active.
+        """
+        self.ag.center_of_geometry(pbc=True)
+
+    def time_moment_of_inertia_default(self, num_atoms):
+        """Benchmark moment_of_inertia calculation with
+        pbc and unwrap inactive.
+        """
+        self.ag.moment_of_inertia()
+
+    def time_moment_of_inertia_pbc(self, num_atoms):
+        """Benchmark moment_of_inertia calculation with
+        pbc active.
+        """
+        self.ag.moment_of_inertia(pbc=True)
+
+    def time_asphericity_default(self, num_atoms):
+        """Benchmark asphericity calculation with
+        pbc and unwrap inactive.
+        """
+        self.ag.asphericity()
+
+    def time_asphericity_pbc(self, num_atoms):
+        """Benchmark asphericity calculation with
+        pbc active.
+        """
+        self.ag.asphericity(pbc=True)
+
+
+class AtomGroupMethodsBenchWithUnwrap(object):
+    """Benchmarks for the various MDAnalysis
+    atomgroup attributes using unwrap.
+    """
+
+    params = (10, 100, 1000, 10000)
+    param_names = ['num_atoms']
+
+    def setup(self, num_atoms):
+        self.u_unwrap = mda.Universe(TRZ_psf, TRZ)
+        self.ag_unwrap = self.u_unwrap.residues[0:3]
+
+        if MDAnalysis.__version__ < '0.20':
```

Comment:
Let's not rely on simple ordering, follow https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python
```python
from packaging import version
if version.parse(MDAnalysis.__version__) < version.parse("0.20.0"):
   ...
```

(In our `setup.py` we use `LooseVersion()` but it looks as if this is pretty much deprecated. If the above does not work, maybe look at the other answer https://stackoverflow.com/a/21065570 using `from pkg_resources import parse_version`, which is guaranteed to assist)